packages/@pollyjs/core/tests/unit/-private/container-test.js

```
@@ -27,16 +27,20 @@ describe('Unit | Container', function() {
 
     it('.register()', function() {
       container.register(Factory);
-      expect(container.has('factory:foo')).to.be.true;
+      expect(container.has('factory:factory-id')).to.be.true;
     });
 
     it('.register() - validation', function() {
-      class NoName extends Factory {
+      class NoId extends Factory {
         /* eslint-disable-next-line getter-return */
-        static get name() {}
+        static get id() {}
       }
 
       class NoType extends Factory {
+        static get id() {
```

masked exception since constructor.name was always a thing